ui/apps/platform/src/types/table.ts

```diff
@@ -1,20 +1,35 @@
 import { ReactElement } from 'react';
+import * as yup from 'yup';
 
 export type { ThProps } from '@patternfly/react-table';
 
-export type SortDirection = 'asc' | 'desc';
-export type AggregateFunc = 'max' | 'count' | 'min';
+export const SortDirectionSchema = yup.string().oneOf(['asc', 'desc']).defined();
+export const AggregateFuncSchema = yup.string().oneOf(['max', 'count', 'min']).defined();
+export const SortAggregateSchema = yup.object({
+    aggregateFunc: AggregateFuncSchema.required(),
+    distinct: yup.string().oneOf(['true', 'false']).optional(),
```

Comment:
Find in Files for `distinct` has 10 results in 6 files, which seem evenly divided:
* string initialization in CVEsTable.tsx and sortUtils.tsx, declaration and validation here, but unclear where it is used
* boolean for search.ts, useURLSort.ts where I see `!!distinct`, and its unit test

Reply:
👀  Good catch, I'll double check this and add some more unit tests.